src/Mod/TechDraw/Gui/TaskProjGroup.cpp

```diff
@@ -379,6 +379,11 @@ void TaskProjGroup::turnProjGroupToView()
     view = viewPart;
     multiView = nullptr;
 
+    auto* vpView = static_cast<ViewProviderViewPart*>(activeGui->getViewProvider(viewPart));
+    if (vpView) {
+        vpView->fixSceneDependencies();
+    }
+
```

Comment:
https://github.com/FreeCAD/FreeCAD/blob/9aa27d7dc3d80aa86daf361d44a43e50e4617ebc/src/Mod/TechDraw/Gui/TaskProjGroup.cpp#L373-L385

No need to cast twice, suggestion:

```cpp
    viewPart->recomputeFeature();
    auto* vp = static_cast<ViewProviderProjGroupItem*>(activeGui->getViewProvider(viewPart));
    if (vp) {
        vp->updateIcon();
        vp->fixSceneDependencies();
    }
    view = viewPart;
    multiView = nullptr;
```